ext/pdo/tests/bug_36798.phpt

```diff
@@ -26,18 +26,24 @@ require_once getenv('REDIR_TEST_DIR') . 'pdo_test.inc';
 $db = PDOTest::factory();
 
 @$db->exec("SET NAMES 'LATIN1'"); // needed for PostgreSQL
-$db->exec("CREATE TABLE test (id INTEGER)");
-$db->exec("INSERT INTO test (id) VALUES (1)");
+$db->exec("CREATE TABLE test36798 (id INTEGER)");
+$db->exec("INSERT INTO test36798 (id) VALUES (1)");
 
-$stmt = $db->prepare("SELECT '�' as test FROM test WHERE id = :id");
+$stmt = $db->prepare("SELECT 'Ã' as test FROM test36798 WHERE id = :id");
 $stmt->execute(array(":id" => 1));
 
 $row = $stmt->fetch(PDO::FETCH_NUM);
 var_dump( $row );
 
+?>
+--CLEAN--
+<?php
+require_once getenv('REDIR_TEST_DIR') . 'pdo_test.inc';
+$db = PDOTest::factory();
+PDOTest::dropTableIfExists($db, "test36798");
 ?>
 --EXPECT--
 array(1) {
   [0]=>
-  string(1) "�"
+  string(2) "Ã"
```

Comment:
I'm confused by this change now

Reply:
Ah, never mind, I was ~stupid~ lazy until now.. Now, I tried to edit the file in TextEdit, but I didn't have luck yet.. So I'll continue to try to get rid of the change....

Reply:
@kocsismate Have you tried `git checkout master -p -- ext/pdo/tests/bug_36798.phpt`?

Reply:
@iluuu1994 thanks for the idea, it looks like it worked! Previously, I tried to checkout master, copy-paste the original code, and add my changes in TextEdit, but somehow it didn't work -- I'm not sure where I messed up.